viz.yaml

```diff
@@ -279,7 +277,7 @@ publish:
       line1: "Streamflow: U.S. Geological Survey National Water Information System ("
       waterDataLink: http://waterdata.usgs.gov/nwis
       line2: >-
-        ). Time series of discharge at gages within 150 km of the storm cone are shown through time, 
+        ). Time series of discharge at gages within 150 km of the eye of the storm are shown through time, 
         ordered by latitude and normalized to site-specific maximum discharge observed over the duration of the hurricane. 
       line4: "Precipitation: National Weather Service Environmental Modeling Center National Center for Environmental Prediction Stage IV precipitation analysis ("
       emcLink: http://www.emc.ncep.noaa.gov/mmb/ylin/pcpanl/stage4/
```

Comment:
the thumbnail needs to be added to the yaml to export it. Not sure how that works. @wdwatkins do you know?